contractcourt/briefcase.go

```diff
@@ -341,6 +349,11 @@ var (
 	// store the anchor resolution, if any.
 	anchorResolutionKey = []byte("anchor-resolution")
 
+	// breachResolutionKey is the key under the logScope that we'll use to
+	// store the breach resolution, if any. This is used rather than the
+	// resolutionsKey.
```

Comment:
Using the resolutionsKey would mean that we'd have to store the breach resolution after the commit & htlc resolutions and since we're probably going to have to upgrade the breach resolutions anyways in a later pr when the breacharbiter logic lives in the breachresolver, it seemed easier to have its own db key